### ISSUE TITLE
geom_column misreguarded by _postgis_stats call

### ISSUE THREAD

**Issue by strk** [CartoDB/Windshaft#504]
Given this MapConfig input:
```
{
  "version": "1.5.0",
  "layers": [
    {
      "type": "mapnik",
      "options": {
        "sql": "select * from liberia",
        "geom_column": "geom",
        "cartocss": "#s { marker-width: 10; }",
        "cartocss_version": "2.0.0",
        "interactivity": "instance"
      }
    }
  ]
}
```

Windshsaft makes a bogus SQL call:
```
2016-11-09 16:07:20 UTC [190-1] akvo@akvo ERROR:  attribute "the_geom" does not exist
2016-11-09 16:07:20 UTC [190-2] akvo@akvo STATEMENT:  SELECT row_to_json(s) as result FROM(select _postgis_stats('public.liberia'::regclass, 'the_geom') as stats) as s
```

The hard-coded `the_geom` name is in lib/windshaft/utils/query_utils.js, function "getTableStats"

**Comment by strk** [comment]
The call to the PostGIS-private `_postgis_stats` is performed for each table in the set of "affected tables" (this happens in `lib/windshaft/metadata/mapnik-layer-metadata.js`, function `MapnikLayerMetadata.prototype.getMetadata`) but this is wrong, in that not all affected tables necessarely have a geometric field, while _postgis_stat (which should not even be called, being an internal function)  fetches stats for a specific geometry column.

Beside, `_postgis_stat` was introduced in PostGIS 2.1.0, while the dependency in README.md mentions PostGIS 1.5.x

**Comment by strk** [comment]
The `getMetadata` function in MapnikLayerMetadata.prototype, according to Renderers API (https://github.com/CartoDB/Windshaft/blob/master/lib/windshaft/renderers/README.md) is supposed to retrieve metadata for a `tileset`, but format of the metadata is said to be documented in the `renderer documentation`, which I cannot find. Can you help with that ?

**Comment by rochoa** [comment]
This is fixed at 549f35b, where the `geom_column` is used for computing stats.

Also, a58f7e7 turns off layer metadata stats by default.

**Comment by strk** [comment]
Thanks! Any chance to deal with #506 too ?
